### PR TITLE
Fixing the failure of parameter name for invalid mode in eventwaithandle

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

File: src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public partial class EventWaitHandle : WaitHandle
         public EventWaitHandle(bool initialState, EventResetMode mode, string name, out bool createdNew)
         {
             if (mode != EventResetMode.AutoReset && mode != EventResetMode.ManualReset)
-                throw new ArgumentException(SR.Argument_InvalidFlag, nameof(mode));
+                throw new ArgumentException(SR.Argument_InvalidFlag);
```
Comment (jkotas) [inline src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs]: Should we rather fix the test?

Comment (Anipik) [reply src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs]: Do we want the mode to be in the exception ?

Comment (jkotas) [reply src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs]: We typically include the argument name in the exception message.

Comment (Anipik) [reply src/System.Private.CoreLib/shared/System/Threading/EventWaitHandle.cs]: okay I will edit the test to include the ```mode``` parameter. But then we will have to make the change when the maestro brings the new version of coreclr into corefx

**Pull request closed without merging**